### DISCUSSION TITLE
Update NetBox Docker v0.13.0 to v1.2.0

### DISCUSSION THREAD

**Discussion by nico586** [netbox-community/netbox-docker#549] [Getting Help] [answered]
Hello everyone,

I need your help to update my NetBox Docker version. I tried a lot of things to make this upgdate but it still doesn't work. So, can you walk me through the steps to go from Docker version 0.13.0 to version 1.2.0 ?

I tried this : https://github.com/netbox-community/netbox-docker/wiki/Updating   but it doesn't work.

Thanks

**Comment by ryanmerolle** [comment, marked as answer]
Did you try https://github.com/netbox-community/netbox-docker/wiki/Updating#common-issues ?

Also be sure to read through the release notes of netbox and netbox-docker for each major version you traverse given there could be some gotchas they point out in the release notes.

Big jumps are sometimes hard to do, and this is exactly why I try to stay close a recent release.
